docs/1.8.9/chattrigers.md

```diff
@@ -0,0 +1,191 @@
+# ChatTrigers Alternatives
+
+ChatTriggers to run the mod which can cause performance issues. These are all common modules I have found people.
+
+List is a WIP but should have all features just is missing some alternatives. Feel free to make a pull request to add them.
+
+I do not have ChatTrigers and don't intend on downloading it. If you know have any more info please pull request it.
```

Comment:
Why…
Just make a test instance and download it… module pages are probably outdated

Reply:
Turns out I can just download the code form the modules page so I don't need to set it up